### PR TITLE
Databases/week2

### PR DESCRIPTION
**Pull request by GreeshmaRahool**
Finished Databases/week2 and waiting for the feedback.

### CODE REVIEW

File: databases/week2/Part3.sql [hunk 1]
```diff
@@ -0,0 +1,46 @@
+--Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT task.title AS Task,user.email
+FROM task 
+JOIN user_task 
+ON task.id = user_task.task_id
```
Comment (sboutzen) [inline databases/week2/Part3.sql:5]: I would keep the ON on the same line as the JOIN, imho it reads better.

File: databases/week2/Part2.sql [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+   `name` VARCHAR(255) NOT NULL,
+   `email`VARCHAR(255) NOT NULL,
+   `phone`VARCHAR(255) NULL,
+   `class_id` INT(10) UNSIGNED NOT NULL,
+    CONSTRAINT `fk_class` FOREIGN KEY (`class_id`) 
+    REFERENCES `class` (`id`) 
+    ON DELETE CASCADE ON UPDATE CASCADE
+ )ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+  --Create an index on the name column of the student table.
+  CREATE INDEX idx_name
```
Comment (sboutzen) [inline databases/week2/Part2.sql:31]: No need for a new line here. Same with the ALTER TABLE statement. When a statement is not very long, often times it is simply easier to read if you put it all in one line. Add newline when it adds readability

**Comment by sboutzen** [comment]
Small comments, well done! 💪

**Comment by sboutzen** [comment]
Very clean commit history btw :)

File: databases/week2/Part3.sql (hunk 1 above)
Comment (GreeshmaRahool) [reply databases/week2/Part3.sql:5]: I was totally confused while formatting queries. Thanks for the comments.

**Pull request merged**